independent-publisher-connectors/Safaricom Mpesa Daraja/apiDefinition.swagger.json

```diff
@@ -0,0 +1,620 @@
+                "ResponseDescription": {
+                  "type": "string",
+                  "description": "ResponseDescription"
+                }
+              }
+            }
+          }
+        },
+        "summary": "make a B2C payment",
+        "description": "make a B2C payment",
+        "operationId": "B2Cpayment",
+        "parameters": [
+          {
+            "name": "Content_Type",
+            "in": "header",
+            "required": false,
+            "type": "string",
+	    "description": "make a B2C payment",
+            "x-ms-summary": "make a B2C payment"
+
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "InitiatorName": {
+                  "type": "string",
+                  "description": "InitiatorName"
+                },
+                "SecurityCredential": {
+                  "type": "string",
+                  "description": "SecurityCredential"
+                },
+                "CommandID": {
+                  "type": "string",
+                  "description": "CommandID"
+                },
+                "Amount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Amount"
+                },
+                "PartyA": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "PartyA"
+                },
+                "PartyB": {
+                  "type": "integer",
+                  "format": "int64",
+                  "description": "PartyB"
+                },
+                "Remarks": {
+                  "type": "string",
+                  "description": "Remarks"
+                },
+                "QueueTimeOutURL": {
+                  "type": "string",
+                  "description": "QueueTimeOutURL"
+                },
+                "ResultURL": {
+                  "type": "string",
+                  "description": "ResultURL"
+                },
+                "Occassion": {
+                  "type": "string",
+                  "description": "Occassion"
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2-auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "tokenUrl": "https://daraja.onrender.com/generate",
+      "scopes": {
+        "User.Read": "User.Read"
+      }
+    }
+  },
+  "security": [
+    {
+      "oauth2-auth": [
+        "User.Read"
+      ]
+    }
+  ],
+  "tags": [],
+   "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.developer.safaricom.co.ke/"
```

Comment:
@johnmuchiriM this should be a valid URL. 
![image](https://user-images.githubusercontent.com/97551789/230199095-a3e17057-aa2f-4056-9490-ec2636da7802.png)